test/integration_tests/long/test_model_train.py

```diff
@@ -72,6 +72,23 @@ def test_ludwig_automl(self):
         self.assertEqual(len(result.columns), 1)
         self.assertEqual(len(result), 10)
 
+    def test_sklearn_regression(self):
+        create_predict_function = """
+            CREATE FUNCTION IF NOT EXISTS PredictHouseRent FROM
+            ( SELECT number_of_rooms, number_of_bathrooms, days_on_market, rental_price FROM HomeRentals )
+            TYPE Sklearn
+            PREDICT 'rental_price'
+            TIME_LIMIT 120;
```

Comment:
remove TIME_LIMIT. It is not used for Sklearn.

Reply:
Ohh okay, removed.